modules/appeals_api/config/locales/en.yml

```diff
@@ -14,4 +14,8 @@ en:
         no_evidence_submission_accepted: "Corresponding Notice of Disagreement 'boardReviewOption' must be 'evidence_submission'"
         not_found: "%{type} with uuid %{id} not found"
         not_homeless_address_missing: "If not homeless, address must be provided: '/data/attributes/veteran/address'"
```

Comment:
What do we think of these error messages? Happy to update anything here

Reply:
I'd have UX take a look at the wording just to be safe - I recall there's some preferred wording around those experiencing homelessness that's used.